### ISSUE TITLE
Project classes are missing in classpath

### ISSUE THREAD

**Issue by anatoly-shuvalov** [openapi-tools/swagger-maven-plugin#6]
Hello,

I have a weird issue when trying to use the plugin. I created a new project using one of the unit-tests poms (generate-mojo-full-pom.xml + dependencies) and sample classes (io.openapitools.swagger.*). However, when I'm trying build a project it generates an empty open-api.yaml:

```
openapi: 3.0.1
info:
  title: Title
  description: |-
    # Description
    This is some description to be included.
  termsOfService: Terms
  contact:
    name: My Name
    url: https://google.com
    email: e@mail.com
  license:
    name: MIT
    url: https://license
  version: 1.0.0
servers:
- url: https://services.example.it/base/path
  description: Example URL
```

I spent some time debugging the plugin and narrowed it down to io.openapitools.swagger.JaxRSScanner - looks like project classes are simply missing in the classpath despite compilation phase is passed already.

Maven also doesn't run the plugin by default unless I either add an "execution" section in the pom file or call it explicitly:
`mvn clean package io.openapitools.swagger:swagger-maven-plugin:generate`

Plugin's integration tests work fine for me - project classes are in the classpath.

I use macOS, Maven version is 3.5.4

**Comment by langecode** [comment]
Thanks for reporting.

I think the classpath issue is a variation of #5. Even though I have used the previous version of the plugin for production it seems my setups have always been including the module itself as dependency of the plugin. I am working on a fix for this.

With respect to execution you will need to include the execution section - however you can leave out the “phase” as this can be defaulted in the mojo and is defaulted to prepare-package. I will correct the documentation though.

**Comment by langecode** [comment]
Class path issue should be fixed by release [2.0.1](https://github.com/openapi-tools/swagger-maven-plugin/tree/v2.0.1).
